src/main/scala-2.11/org/apache/james/gatling/control/JamesWebAdministration.scala

```diff
@@ -0,0 +1,18 @@
+package org.apache.james.gatling.control
+
+import play.api.libs.ws.ning.NingWSClient
+
+object JamesWebAdministration {
+
+  val wsClient = NingWSClient()
+
+  val baseUrl: String = "http://172.17.0.4:8000"
+
+  def addDomain(domain: String) = wsClient.url(s"$baseUrl/domains/$domain")
```

Comment:
in scala, you have value types, please use them (and define a Domain object, a User object, etc), it's so cheap to do that you don't have any excuse not to use them